ui/src/App.re

```diff
@@ -1,21 +1,22 @@
-module LoginScreen = {
-  [@bs.module "./deprecated/login.js"] [@react.component]
-  external make: unit => React.element = "default";
+module AuthenticatedRoute = {
```

Comment:
Just a FYI - I've made it so that any route that requires login should be wrapped in this component. Eg the "about" page probably doesn't need to be wrapped in it.

This component is basically just the "Login" component, maybe I should delete it and just use `<Login>` directly instead.